fetch-mock/fetch-mock.d.ts

```diff
@@ -0,0 +1,144 @@
+// Type definitions for fetch-mock 4.6.0
+// Project: https://github.com/wheresrhys/fetch-mock
+// Definitions by: Alexey Svetliakov <https://github.com/asvetliakov>, Tamir Duberstein <https://github.com/tamird>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+/// <reference path="../whatwg-fetch/whatwg-fetch.d.ts" />
+
+
+declare module "fetch-mock" {
+    /**
+     * Mock matcher function
+     * @param url
+     * @param opts
+     */
+    type MockMatcherFunction = (url: string, opts: Object) => boolean;
```

Comment:
Do not use the `Object` type. Use either `{}` or `any` instead. Here `{}` may be more appropriate